integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/MicroRegistryManager.java

```diff
@@ -123,19 +123,26 @@ public void updateResource(String resourcePath, String sourcePath, boolean backu
     /**
      * Function is to check if the registry resources are exist in filepath
      *
-     * @param sourcePath path to registry resource
      * @param resourcePath path to resource file path from registry resource
      * @param filename registry filename
      */
 
-    public boolean checkResourceExist(String sourcePath, String resourcePath, String filename) throws Exception {
-        try {
-            String path = sourcePath + resourcePath + filename;
-            return path.contains(filename);
+    public boolean checkResourceExist(String resourcePath, String filename) throws Exception {
```

Comment:
Do we need to throw a generic exception?

Reply:
Done